static/css/dashboard.css

```diff
@@ -64,8 +65,9 @@
   flex-direction: column;
   align-items: center;
   height: 100%;
-  background-color: var(--db-options-bg-color);
   border-radius: 5px 0 0 0;
+  border: 1px solid #DCDBDF;
```

Comment:
You should use variables to define colors for both the dark and light themes. This way, you won't need to redefine the styles for the dark theme separately.